lib/receipt.rb

```diff
@@ -0,0 +1,29 @@
+require 'json'
+
+class Receipt
+  include Product_list
+
+  def header
+    {shopName: data_hash[0]['shopName'], address: data_hash[0]['address'], phone: data_hash[0]['phone']}
```

Comment:
Line is too long. [104/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.